editor/scene_tree_dock.cpp

```diff
@@ -330,6 +331,73 @@ void SceneTreeDock::_perform_instantiate_scenes(const Vector<String> &p_files, N
 	}
 }
 
+void SceneTreeDock::_perform_create_audio_stream_players(const Vector<String> &p_files, Node *parent, int p_pos) {
+	ERR_FAIL_NULL(parent);
+
+	StringName node_type = "AudioStreamPlayer";
+	if (Input::get_singleton()->is_key_pressed(Key::SHIFT)) {
+		if (Object::cast_to<Node2D>(parent)) {
+			node_type = "AudioStreamPlayer2D";
+		} else if (Object::cast_to<Node3D>(parent)) {
+			node_type = "AudioStreamPlayer3D";
+		}
+	}
+
+	Vector<Node *> nodes;
+	bool error = false;
+
+	for (int i = 0; i < p_files.size(); i++) {
+		Ref<AudioStream> stream = ResourceLoader::load(p_files[i]);
+		if (stream.is_null()) {
+			current_option = -1;
+			accept->set_text(vformat(TTR("Error loading audio stream from %s"), p_files[i]));
+			accept->popup_centered();
+			error = true;
+			break;
+		}
+
+		Node *player = Object::cast_to<Node>(ClassDB::instantiate(node_type));
+		player->set("stream", stream);
+		player->set_name(Node::adjust_name_casing(p_files[i].get_file().get_basename().validate_node_name()));
```

Comment:
This can result in empty name, e.g. when file is named `%.mp3`. It's a corner-case, but results in an error.

Reply:
Dropping texture as Sprite2D has a similar issue for `_.mp3`. I'll try to fix both.